src/test/java/lotto/domain/WinningLottoTest.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class WinningLottoTest {
+
+    @Test
+    void 생성자_정상입력_생성성공() {
+        assertThatCode(() -> new WinningLotto(new Lotto(1, 2, 3, 4, 5, 6), new LottoNumber(7)))
```

Comment:
```suggestion
        assertThatCode(() -> new WinningLotto(new Lotto(1, 2, 3, 4, 5, 6), 7))
```
int 부 생성자 추가해 위와 같이 구현 가능하지 않을까?